test/test_parser.cpp

```diff
@@ -3025,15 +3025,15 @@ TEST(Parser_Backlinks)
 
     // items purchased more than twice
     verify_query(test_context, items, "@links.class_Person.items.@count > 2", 2);
-    verify_query(test_context, items, "@LINKS.class_Person.items.@size > 2", 2);
+    verify_query(test_context, items, "@links.class_Person.items.@size > 2", 2);
```

Comment:
I was going to point out that we can make this case insensitive, but actually I think it's reasonable to require these types of operators as all lowercase as long as we are consistent. 👍